### DISCUSSION TITLE
2.4版本后需要挂梯子才能加载出来是什么情况

### DISCUSSION THREAD

**Discussion by Majjcom** [the1812/Bilibili-Evolved#3701] [提问 / Q&A]
2.4版本后需要挂梯子才能加载出来是什么情况

更新后还得挂梯子才能加载（虽然一次加载完就暂时没啥问题了）

有没有不用梯子的解决方案

**Comment by the1812** [comment]
![image](https://user-images.githubusercontent.com/26504152/191978754-b5b7bbbd-2d35-480f-b1a1-3dfba54ab29e.png)
